packages/discord.js-utilities/src/lib/utilities.ts

```diff
@@ -1,5 +1,6 @@
 import { isNullish, Nullish } from '@sapphire/utilities';
 import { Permissions } from 'discord.js';
+import { isThreadChannel } from '..';
```

Comment:
Eek. This import makes me die inside (and I've been avoiding them because I'd rather they be explicit.. idk what you want tho)